frontend/app/assets/javascripts/linker.js

```diff
@@ -416,6 +434,14 @@ $(function() {
             enableSorting();
             $linkerWrapper.addClass("sortable");
           }
+
+          // This is part of automatically executing a search for the current resource on the browse top containers modal when opened from the edit resource page.
+          let currentPath = window.location.pathname
+          // If this setTimeout is for the last linker in the modal, only then is it safe to execute the search
+          let lastLinker = $(".modal-dialog").find(".linker").last()
+          if (lastLinker.attr("id") === $this.context.id && currentPath.match(/^\/resources\/[\s\S]*\/edit$/) && $(".label.label-info").text() === "Resource" && $(".modal-dialog").find("#collection_resource").length > 0 && $(".modal-dialog").find(".table-search-results").length < 1) {
```

Comment:
That's a pretty long line of conditionals.  I'm not insisting on it, but I'd be strongly in favor of breaking them down into named vars for readability (e.g. `on_resource_edit_path`, `id_matches`, `has_resource`, `results_empty`).  

Also change the regex here as well, per previous comment; in fact, that regex can be stored in a var and reused, or potentially it might even make sense to move the check for path up to a higher scope and reuse the result of the check in both places.